### PR TITLE
[TC2-1296] Latvia documentation

### PR DESCRIPTION
**Pull request by kalmerkaurson**

### CODE REVIEW

File: affiliates/latvia.swagger.json [hunk 1]
```diff
@@ -0,0 +1,533 @@
...
...api.credy.eu/v3/)\n\nStaging URL: [http://api.staging.credy.eu/v3/](http://api.staging.credy.eu/v3/)\n\nAPI supports post fields, XML and JSON.\n\nTo get response error messages in Latvian, \"Accept-Language\" header’s value must be set to according ISO code. Ex: `lv` or `lv-LV`.\n## Authorization\nAuthorization is performed with OAuth 2.0 Client Credentials Grant.\nWhen requesting token, the client key and secret key must be sent in request body.\nClient key and secret key will be assigned to you by Credy.\nThe requests to api must be authenticated with [bearer token](https://tools.ietf.org/html/rfc6749#section-7.1).\n## Lead registration flow\n\n1. Request is sent to /v3/customers\n\n2. If request is successful, customer uuid must be stored\n\n3. Request is sent to /v3/leads with previously stored customer uuid in query parameter \"customer\" *ex. /v3/leads?customer=a3ff7c20-e9e8-43a0-bfb7-71a5cf43b8d3* \n4. When the lead is accepted by api (under_investigation is received from POST lead), please redirect the customer to [https://www.credy.lv/loading/?lead={UUID}](https://www.credy.lv/loading/?lead={UUID}) (where {UUID} is replaced by the uuid that you receive from POST lead)\n",
```
Comment (mikk150) [inline affiliates/latvia.swagger.json]: "This document further describes **fdfhow** to integrate"

**Review by Nuffic — APPROVED** [review]

**Review by mikk150 — CHANGES_REQUESTED** [review]

File: affiliates/latvia.swagger.json (hunk 1 above)
Comment (kalmerkaurson) [reply affiliates/latvia.swagger.json]: how indeed

File: affiliates/latvia.swagger.json [hunk 2]
```diff
@@ -0,0 +1,533 @@
...
+    },
+    "schemas" : {
+      "PrimaryAddress" : {
+        "required" : [ "city", "street", "postal_code" ],
+        "type" : "object",
+        "properties" : {
+          "postal_code" : {
+            "type" : "string",
+            "description" : "Format: /^\\d{4}$/",
+            "example" : "38627",
```
Comment (mikk150) [inline affiliates/latvia.swagger.json]: description format sais 4 numbers, but example has 5?

Comment (kalmerkaurson) [reply affiliates/latvia.swagger.json]: fixed the example

**Review by mikk150 — APPROVED** [review]

**Pull request merged**